platform-coop-toolkit/coop-map-directory-index
# feat: replace category groups with types, surface type in admin (resolves #94)

## Description

This PR replaces category groups with a ForeignKey relationship to types as defined in #94.

## Steps to test

1. Check out branch.
2. Run migration.
3. View categories and types in admin.

**Expected behavior:** 

1. Category "Category type" should list available Types.
2. Types should be editable.

## Additional information

Follow-up tasks once merged:

1. Add missing types.
2. Update categories to the appropriate category type.
3. More to follow.

## Related issues

- Resolves #94.
